app/lib/Plugins/SearchEngine/SqlSearch.php

```diff
@@ -863,79 +871,86 @@ private function _doQueriesForSqlSearch($po_rewritten_query, $pn_subject_tablenu
+                                    $vs_sql = "
 									INSERT INTO {$vs_temp_table}
 									SELECT swi.index_id + 1, 1, null
 									FROM ca_sql_search_words sw 
 									INNER JOIN ca_sql_search_word_index AS swi ON sw.word_id = swi.word_id 
-									".(sizeof($va_temp_tables) ? " INNER JOIN ".$va_temp_tables[sizeof($va_temp_tables) - 1]." AS tt ON swi.index_id = tt.row_id" : "")."
+									" . (sizeof($va_temp_tables) ? " INNER JOIN " . $va_temp_tables[sizeof($va_temp_tables) - 1] . " AS tt ON swi.index_id = tt.row_id":"") . "
 									WHERE 
 										sw.word = ? AND swi.table_num = ? {$vs_fld_limit_sql}
- 										".($this->getOption('omitPrivateIndexing') ? " AND swi.access = 0" : '')."
+ 										" . ($this->getOption('omitPrivateIndexing') ? " AND swi.access = 0":'') . "
 								";
-								$qr_res = $this->opo_db->query($vs_sql, $vs_word, (int)$pn_subject_tablenum);
-								
-								$qr_count = $this->opo_db->query("SELECT count(*) c FROM {$vs_temp_table}");
-								
-								$va_temp_tables[] = $vs_temp_table;	
-							}
-							
-							$vs_results_temp_table = array_pop($va_temp_tables);
-							
-							$this->opo_db->query("UPDATE {$vs_results_temp_table} SET row_id = row_id - 1");
-							$va_direct_query_temp_tables[$vs_results_temp_table] = true;
-							$vs_direct_sql_query = "SELECT swi.row_id, ca.boost, ca.field_container_id
+                                    $qr_res = $this->opo_db->query($vs_sql, $vs_word, (int) $pn_subject_tablenum);
+
+                                    $qr_count = $this->opo_db->query("SELECT count(*) c FROM {$vs_temp_table}");
+
+                                    $va_temp_tables[] = $vs_temp_table;
+                                }
+
+                                $vs_results_temp_table = array_pop($va_temp_tables);
+
+                                $this->opo_db->query("UPDATE {$vs_results_temp_table} SET row_id = row_id - 1");
+                                $va_direct_query_temp_tables[$vs_results_temp_table] = true;
+                                $vs_direct_sql_query = "SELECT swi.row_id, ca.boost, ca.field_container_id
 													FROM {$vs_results_temp_table} ca
 													INNER JOIN ca_sql_search_word_index AS swi ON swi.index_id = ca.row_id 
-							";
-							$pa_direct_sql_query_params = array(); // don't pass any params
-							
-							foreach($va_temp_tables as $vs_temp_table) {
-								$this->_dropTempTable($vs_temp_table);
-							}
-							
-							break;
-		}
+							        ";
+                                $pa_direct_sql_query_params = array(); // don't pass any params
+
+                                foreach ($va_temp_tables as $vs_temp_table) {
+                                    $this->_dropTempTable($vs_temp_table);
+                                }
+
+                                break;
+                            }
+                            // TODO: Missing break statement
```

Comment:
It intentionally falls to the default case here when strictPhraseSearching is not in effect.